.github/workflows/weekly_release.yml

```
@@ -40,9 +40,7 @@ jobs:
           branch: "weekly-release-${{ env.CRATE_VERSION }}"
           commit-message: "[weekly-release] ${{ env.CRATE_VERSION }}"
           title: "Release ${{ env.CRATE_VERSION }}"
-          labels: |
-            "⛴ release"
-            "exclude from changelog"
+          labels: "⛴ release, exclude from changelog"
```

```suggestion
          labels: |
            ⛴ release
            exclude from changelog
```
YAML moment, I think this should work if we just remove the quotes

---

Just guessing based on what the output of the action was in the previous run https://github.com/rerun-io/rerun/actions/runs/5927259856/job/16070405253#step:6:7

---

god damn I hate yaml

---

But yeah, I think you are right: https://github.com/peter-evans/create-pull-request#reference-example